src/style.css

```
@@ -0,0 +1,108 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.form {
+  position: relative;
+  display: flex;
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  bottom: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.icon--small {
+  top: 15px;
+  left: 13px;
+  bottom: 16px;
+  width: 11px;
+  height: 11px;
+}
+
+.input {
+  width: 100%;
+  font-family: Avenir-Heavy, sans-serif;
```

please add the font-family style for whole `html` selector. here you can just use:
```suggestion
  font-family: inherit;
```